src/components/login/Landing.vue

```diff
@@ -64,6 +65,12 @@ export default {
       }
     }
   }
+
+  .support-links {
+    margin-top: 30px;
```

Comment:
You can use:
`margin-top: toRem(30px);`

Reply:
Fixed